docs/_data/prague-2021-schedule.yaml

```diff
@@ -0,0 +1,79 @@
+writing_day:
+- time: '10:00'
+  title: 'Platform opens'
+- time: '11:00'
+  title: Morning introduction - Introduction to Sprints & Project Introductions
+- time: '11:15'
+  title: Team up and start working
+- time: '14:30'
+  title: Lunch Break
+- time: '19:00'
+  title: End of Writing Day
+
+talks_day1:
+- time: '9:00'
```

Comment:
@ddbeck @bethaitman when do you want to do the second WW tour? In Portland we had it at :15 past the opening hour and it ran for about 30min